src/features/user/Account/CancelModal.tsx

```diff
@@ -147,25 +150,17 @@ export const CancelModal = ({
               {showCalender ? (
                 <>
                   <ThemeProvider theme={materialTheme}>
-                    <MuiPickersUtilsProvider
-                      utils={DateFnsUtils}
-                      // locale={
-                      //   localeMapForDate[userLang]
-                      //     ? localeMapForDate[userLang]
-                      //     : localeMapForDate['en']
-                      // }
-                    >
-                      <Calendar
+                    <LocalizationProvider dateAdapter={AdapterDateFns}>
```

Comment:
Calendar - font colours for selected date should be white. Also, the colour is different for the initially selected date, and when the value is changed. Perhaps that should be set as well.

Reply:
@mohitb35 MUI-date-pickers comes from [@mui/lab](https://www.npmjs.com/package/@mui/lab).
Components in Mui lab are not completely stable. So for now Mui Core doesn't provide a standard way to style these components.

Reply:
We don't explicitly have to import it from MUI X. Importing from core does that for us.

![image](https://user-images.githubusercontent.com/37466979/167296399-292f4a20-2753-471a-8b09-c8f848404ee3.png)

Reply:
Issue created #1486